### PR TITLE
Added notes related to the quick-fix improvements made in xtext-eclipse/issues/1448

### PR DESCRIPTION
**Pull request by nbhusare**
Signed-off-by: nbhusare <neerajbhusare@gmail.com>

### CODE REVIEW

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 1]
```diff
@@ -13,13 +13,32 @@ Xtext 2.23.0 is a maintenance release.
...
+
 The AbstractCodeMiningTest and the AbstractMultiQuickfixTest base classes have been added to the org.eclipse.xtext.ui.testing package to provide a convenient way to test the code mining and multi-quickfix capabilities. The Xtext example projects and the Xtend code base have been extended by concrete test cases to demonstrate the usage of these framework classes.
 
-## Upgrades
+## Eclipse Integration
 
+### Quickfix improvements
 
-## Removals
+Convert a terminal fragment to a terminal rule. It resolves the following errors
```
Comment (miklossy) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:23]: please add a colon at the end, so `errors:`

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 2]
```diff
@@ -13,13 +13,32 @@ Xtext 2.23.0 is a maintenance release.
...
+
+* A rule call in the parser rule points to a terminal fragment
```
Comment (miklossy) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:25]: please add a dot at the end, so `fragment.`

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 3]
```diff
@@ -13,13 +13,32 @@ Xtext 2.23.0 is a maintenance release.
...
+* A hidden token in the grammar definition or a parser rule points to a terminal fragment.
+
+Remove illegal hidden token definition. It resolves the following errors
```
Comment (miklossy) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:28]: please add a colon at the end, so errors:

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 4]
```diff
@@ -13,13 +13,32 @@ Xtext 2.23.0 is a maintenance release.
...
+
+* The hidden-token points to a parser rule, terminal fragment, or an enum rule.
 
+Create a new rule definition. It resolves the following errors
```
Comment (miklossy) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:32]: please add a colon at the end, so errors:

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 5]
```diff
@@ -13,13 +13,32 @@ Xtext 2.23.0 is a maintenance release.
...
+
+* An undefined parser rule is cross-referenced
```
Comment (miklossy) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:34]: please add a dot at the end, so `cross-referenced.`

**Comment by nbhusare** [comment]
@miklossy Please check now. Tx.

**Comment by miklossy** [comment]
@nbhusare @cdietrich What is the state of the issue https://github.com/eclipse/xtext-eclipse/issues/1448 ? Is it already fixed or is there some work still pending?

**Comment by nbhusare** [comment]
@miklossy It is not fixed yet. The PR is under review.

**Comment by cdietrich** [comment]
thx @nbhusare

**Pull request merged**